### ISSUE TITLE
Calling 'fail' fails...oh the irony

### ISSUE THREAD

**Issue by ediblecode** [nice-digital/wdio-cucumber-steps#37]
In the accessibility tests, if they fail then they call `fail`, which fails. 

>  fail is not defined ReferenceError: fail is not defined at World.checkForAccessibilityIssues (/tests/node_modules/@nice-digital/wdio-cucumber-steps/src/support/check/checkForAccessibilityIssues.ts:85:3)

See https://github.com/nice-digital/wdio-cucumber-steps/blob/dd5f5da83bef716ae38e64aa809593973fb3b9bd/src/support/check/checkForAccessibilityIssues.ts#L85
